lib/test.gi

```diff
@@ -331,12 +331,23 @@ end);
 ##  <List>
 ##  <Mark>#@local identifierlist</Mark>
 ##  <Item>Run all the tests in the input as if it is in a function with local variable list
-##  <C>identifierlist</C>, which is a comma-separated list of
-##  identifers. Multiple #@local lines may be used.
+##  <C>identifierlist</C>, which is a comma-separated list of identifers, 
+##  terminated with a semicolon.  
+##  Multiple <C>#@local</C> lines <E>are</E> allowed,
+##  in which case the final identifier at the end of each line 
+##  except the last should <E>not</E> be followed by a comma. 
```

Comment:
none of the lines should end with a comma

Reply:
Done.